Lib/test/test_c_locale_coercion.py

```diff
@@ -15,7 +15,7 @@
 # Set the list of ways we expect to be able to ask for the "C" locale.
 # 'invalid.ascii' is an invalid LOCALE name and so should get turned in to the
 # default locale, which is traditionally C.
-EXPECTED_C_LOCALE_EQUIVALENTS = ["C", "invalid.ascii"]
+EXPECTED_C_LOCALE_EQUIVALENTS = ["C", "POSIX", "invalid.ascii"]
```

Comment:
The `linux` specific setup clause below about the `POSIX` locale can also be removed given this addition.